odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -271,6 +272,8 @@ def value_to_html(self, value, options):
             format_func = babel.dates.format_time
             return pycompat.to_text(format_func(value, format=pattern, tzinfo=tzinfo, locale=locale))
         if options.get('date_only'):
+            # Make sure the date is in the right timezone
+            value = pytz.utc.localize(value.replace(tzinfo=None)).astimezone(tzinfo)
```

Comment:
I think this is probably incorrect. Why do you need to set tzinfo=None and cast to UTC?
```suggestion
            value = value.astimezone(tzinfo)
```